component/authenticator/src/main/java/org/wso2/carbon/identity/application/authenticator/totp/TOTPAuthenticator.java

```diff
@@ -444,6 +500,26 @@ private void validateAccountLockStatusForLocalUser(AuthenticationContext context
             if (log.isDebugEnabled()) {
                 log.debug(errorMessage);
             }
+            String accountLockedReason = StringUtils.EMPTY;
+            try {
+                UserRealm userRealm = TOTPUtil.getUserRealm(username);
+                UserStoreManager userStoreManager = userRealm.getUserStoreManager();
+                Map<String, String> claimValues = userStoreManager.getUserClaimValues(
+                        IdentityUtil.addDomainToName(authenticatedUserObject.getUserName(),
+                                authenticatedUserObject.getUserStoreDomain()),
+                        new String[]{ TOTPAuthenticatorConstants.ACCOUNT_LOCKED_REASON_CLAIM_URI},
+                        UserCoreConstants.DEFAULT_PROFILE);
+                if (claimValues != null) {
+                    accountLockedReason = claimValues.get(TOTPAuthenticatorConstants.ACCOUNT_LOCKED_REASON_CLAIM_URI);
+                }
+            }
+            catch (UserStoreException e) {
+                throw new AuthenticationFailedException(errorMessage+ " Could not get the account locked reason");
```

Comment:
formatting issue

Reply:
Addressed in 15599e633faceca668e09e0858254946815091ae